### ISSUE TITLE
Add identity pallet and update Polkadot version

### ISSUE THREAD

**Issue by TorstenStueber** [pendulum-chain/pendulum#82]
Please do

- [ ] Add pallet-identity
- [ ] Update to Polkadot version v0.9.33.
- [ ] Incorporate comments from https://github.com/pendulum-chain/pendulum/pull/78

**Comment by vadaynujra** [comment]
@TorstenStueber anything pending on this?

**Comment by vadaynujra** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pendulum-6408a0365cda59c57d39bee5/issues/pendulum-chain/pendulum/82?planning-poker) with Zenhub @b-yap @ebma @TorstenStueber

**Comment by ebma** [comment]
Isn't this done already?

**Comment by TorstenStueber** [comment]
Can be closed. Has been done already long time ago as part of another ticket.
